src/api/db/index.js

```diff
@@ -105,6 +111,23 @@ export default function initBookshelf(config) {
         promises = [...promises, ...morePromises];
       }
 
+      let schools = this.schools();
+      schoolNames = await School.fetchAll();
+      schoolNames = schoolNames.serialize().map(row => row.name);
+
+      for (let name of labelNamesToAdd) {
+        if (schoolNames.indexOf(name) > -1) {
+          schoolNamesToKeep.push(name);
+        }
+      }
+
+      let school_tags = await Promise.all(schoolNamesToKeep.map(tag_name => School.createOrSelect(tag_name)));
+      let schoolPromises = school_tags.map(async (tag) => {
+        await schools.attach(tag)
+      });
+
+      promises = [...promises, ...schoolPromises];
+
```

Comment:
so, you use tag as "label" at first and then as a school-name, in case school-name matches? that's not what was supposed to happen here.

or am I reading code wrong?